integration

```diff
@@ -167,6 +167,7 @@
   "benjamin-dcs/file-plusplus",
   "benleb/sureha",
   "Bennett-Wendorf/hass-uplift-desk",
+  "BenoitDuffez/home-assistant",
```

Comment:
This is just a fork without any changes on top, and should not be added here.
The primary one should be.